apps/app/src/views/routes/Order.tsx

```diff
@@ -62,26 +63,58 @@ export const ORDER_FULFILLMENT_TYPE_MAP = {
   [types.FulfillmentType.MailOrder]: 'Mail order'
 };
 
-export const FILL_STATE_MAP: object = {
-  CANCELED: 'Canceled',
-  NEW: 'New',
-  SCHEDULED: 'Scheduled',
-  SENT: 'Sent'
-};
-export const FILL_COLOR_MAP: object = {
-  CANCELED: 'gray',
-  NEW: 'green',
-  SCHEDULED: 'orange',
-  SENT: 'yellow'
+const CancelOrderAlert = ({
+  orderState,
+  fulfillmentState
+}: {
+  orderState: OrderState;
+  fulfillmentState: OrderFulfillmentState;
+}) => {
+  if (orderState === 'CANCELED') {
```

Comment:
nit: maybe use the order state enums for these?

Reply:
there's a weird issue where it throws an error trying to access the enum, tried to troubleshoot for a while but couldn't debug. Will need to come back to it. I'll add a comment though!

Reply:
looks like you use `types.OrderState.Canceled` below?

Reply:
there's weird stuff going on (this code works btw), going to have to come back to this.

<img width="337" alt="Screen Shot 2023-07-28 at 2 15 32 PM" src="https://github.com/Photon-Health/client/assets/700617/5feadc23-9e75-46c3-ae0b-bcdc7df0189f">